core-v1/src/test/scala/SimpleDivisionSpec.scala

```diff
@@ -0,0 +1,125 @@
+package com.rallyhealth.vapors.v1
+
+import munit.FunSuite
+
+class SimpleDivisionSpec extends FunSuite {
+
+  import dsl.simple._
+
+  // TODO: Should this return an Option[Int]? Maybe another method for this?
+  test("Divide by zero") {
+    val expr = 1.const / 0.const
+    val exc = intercept[ArithmeticException] {
+      expr.run()
+    }
+    assert(exc.getMessage contains "zero")
+  }
```

Comment:
I'm leaving this for a future change. While I don't think expressions should throw exceptions when interpreted. I don't think using an `Option` interface is great either, as there isn't really a way to recover from a divide by zero error any better than just throwing that exception, and there isn't any way to know apriori that the denominator expression would return a `0`.